bokehjs/src/lib/core/ui_events.ts

```diff
@@ -109,17 +137,8 @@ export class UIEvents {
     this.hammer.on('rotatestart', (e) => this._rotate_start(e))
     this.hammer.on('rotate', (e) => this._rotate(e))
     this.hammer.on('rotateend', (e) => this._rotate_end(e))
-
-    this.hit_area.addEventListener("mousemove", (e) => this._mouse_move(e))
-    this.hit_area.addEventListener("mouseenter", (e) => this._mouse_enter(e))
-    this.hit_area.addEventListener("mouseleave", (e) => this._mouse_exit(e))
-
-    this.hit_area.addEventListener("wheel", (e) => this._mouse_wheel(e))
-
-    document.addEventListener("keydown", (e) => this._key_down(e))
-    document.addEventListener("keyup", (e) => this._key_up(e))
   }
-
+  
```

Comment:
You need to remove trailing whitespace here and possibly some other places to fix `codebase` tests.